src/plugins/flow.js

```diff
@@ -380,6 +379,10 @@ pp.flowParseObjectType = function (allowStatic, allowExact) {
       }
       if (this.isRelational("<") || this.match(tt.parenL)) {
         // This is a method property
+        if (variance) {
+          this.unexpected(variance.start);
+          variance = null;
```

Comment:
This doesn't makes sense as `this.unexpected()` throws anyway. Maybe we want to do `delete ...` here like in the other places?